### PR TITLE
LPS-173252 - Automation | Workflow | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
This behavior is being automated because of this bug - [LPS-172352](https://liferay.atlassian.net/browse/LPS-172352)
Automation ticket: [LPS-193162](https://liferay.atlassian.net/browse/LPS-193162)

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
ci:test:workflow

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -1017,6 +1017,35 @@ definition {
...
+		task ("Given that a user accesses the workflow") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Applications",
+				portlet = "Process Builder");
+
+			Workflow.gotoEditWorkflowDefinition(workflowDefinitionTitle = "Single Approver");
+		}
+
+		task ("When changing the notification type") {
```
Comment (patriciajeanperez) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:1032]: @thaiscabrals can we update this task description to specify that it is the recipient type you are changing and not notification type?

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 2]
```diff
@@ -1017,6 +1017,35 @@ definition {
...
 	}
 
+	@description = "LPS-172352 - Verify if it is possible to specify the recipient type in a workflow task"
+	@priority = 4
+	test CanSpecifyTheRecipientTypeInWorkflowTask {
+		task ("Given that a user accesses the workflow") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Applications",
+				portlet = "Process Builder");
```
Comment (patriciajeanperez) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:1027]: @thaiscabrals let's access Process Builder via UI to make our tests faster

Comment (thaiscabrals) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:1027]: Did you mean via API? We're already accessing it via UI.

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase (hunk 2 above)
Comment (patriciajeanperez) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:1027]: sorry I meant URL! @thaiscabrals

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**